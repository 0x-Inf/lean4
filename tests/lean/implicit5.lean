Variable f {A : Type} (a1 a2 : A) : A
Variable g : Int -> Int -> Int
Variable h : Int -> Int -> Real -> Int
Variable p {A B : Type} (a1 a2 : A) (b : B) : A
Infix ++ : f
Infix ++ : g
Infix ++ : h
Infix ++ : p
Variable p2 {A B : Type} (a1 a2 : A) (b : B) {C : Type} (c : C) : A
Infix ++ : p2
