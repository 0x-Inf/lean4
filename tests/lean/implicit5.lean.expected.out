  Set: pp::colors
  Set: pp::unicode
  Assumed: f
  Assumed: g
  Assumed: h
  Assumed: p
  Assumed: p2
The denotation(s) for the existing notation:
  Infix ++
have been replaced with the new denotation:
  @p2
because they conflict on how implicit arguments are used.
